inference-engine/src/inference_engine/include/ie/cldnn/cldnn_config.hpp

```diff
@@ -24,24 +24,27 @@ namespace CLDNNConfigParams {
 /**
  * @brief shortcut for defining configuration keys
  */
-#define CLDNN_CONFIG_KEY(name) InferenceEngine::CLDNNConfigParams::_CONFIG_KEY(CLDNN_##name)
-#define DECLARE_CLDNN_CONFIG_KEY(name) DECLARE_CONFIG_KEY(CLDNN_##name)
+#define CLDNN_CONFIG_KEY(name)           InferenceEngine::CLDNNConfigParams::_CONFIG_KEY(CLDNN_##name)
+#define DECLARE_CLDNN_CONFIG_KEY(name)   DECLARE_CONFIG_KEY(CLDNN_##name)
 #define DECLARE_CLDNN_CONFIG_VALUE(name) DECLARE_CONFIG_VALUE(CLDNN_##name)
 
 /**
- * @brief This key instructs the clDNN plugin to use the OpenCL queue priority hint
```

Comment:
this line is not 120 symbols, why it was split into 2 lines?

Reply:
I didn't find why it is shorter than 120 symbols, clang format doesn't have a separate option for comment limit.
I think `ReflowComments` did this split.

Reply:
Is it possible that you applied clang format with 80 chars limit first and then changed limit to 120? As I can see, clang format don't combine two lines of comments even if they fit new column width.

Reply:
@vladimir-paramuzov Looks like you are right. I have reverted code style fixes and applied it one more time.